File: src/Mobile/MobileApp/scripts/models/moduleconfiguration/LedenetModuleConfiguration.ts
namespace Submerged.Models.ModuleConfiguration {
    "use strict";

    export class LedenetPointInTime {
        time: number;
        fadeIn: number;
        level: number;
        color: string;
        white: number;
    }

    export class LedenetModuleConfigurationModel extends BaseModuleConfiguration {
        device: string;
        pointsInTime: LedenetPointInTime[] = new Array<LedenetPointInTime>();        
    }

    export class LedenetPointInTimeDisplayModel extends LedenetPointInTime {
        timeString: string;
    }


}

File: src/Gateway/Repsaj.Submerged.Gateway.Univeral/Models/ConfigurationModels/LedenetModuleConfiguration.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Repsaj.Submerged.GatewayApp.Universal.Models.ConfigurationModels
{
    public class LedenetModuleConfiguration
    {
        public string Device { get; set; }
        public List<LightingPointInTime> PointsInTime { get; set; }
    }

    public class LightingPointInTime
    {
        public int Time { get; set; }
        public int FadeIn { get; set; }
        public int Level { get; set; }
        public string Color { get; set; }
        public int White { get; set; }
    }
}
